backend/app/main.py

```diff
@@ -1222,3 +1217,57 @@ async def get_survey_results(
         return FileResponse(
             path=_out_file, media_type=media_type, filename=out_file_name
         )
+
+
+@app.get("/api/group/{group_slug}", response_model=MapGroupResponse)
```

Comment:
H: I think we should use the `api/gerrydb/views` endppint with the group argument

Reply:
ok I just pushed a commit for that. I changed the params for `getAvailableDistrictrMaps` and the sort order of the response to be alphabetical; hope those changes makes sense